### PR TITLE
Add removeEncapsulationId API.

### LINKED ISSUE #826: Add removeEncapsulationId API.

### PR DESCRIPTION
**Pull request by hsorby**
Add missing API removeEncapsulationId.

Closes #826.

### CODE REVIEW

File: src/api/libcellml/componententity.h [hunk 1]
```diff
@@ -291,6 +291,13 @@ class LIBCELLML_EXPORT ComponentEntity: public NamedEntity
      */
     std::string encapsulationId() const;
 
+    /**
+     * @brief Remove the encapsulation Id for this entity.
+     *
+     * Clear the encapsulation identifier for this entity.
```
Comment (agarny) [inline src/api/libcellml/componententity.h]: Replace "identifier" with "Id" to be consistent with the rest of the documentation for this class (see `setEncapsulationId()` for instance).

File: src/bindings/interface/componententity.i [hunk 2]
```diff
@@ -77,6 +77,9 @@ element for the model's structure.  For the :class:`Component` class this is
 the ``component_ref`` element that references the component it represents in
 the structure."
 
+%feature("docstring") libcellml::ComponentEntity::removeEncapsulationId
+"Removes the encapsulation id for this entity."
```
Comment (agarny) [inline src/bindings/interface/componententity.i]: "Id" rather than "id".

**Review by agarny — CHANGES_REQUESTED** [review]
Some minor issues otherwise looks good.

File: src/bindings/interface/componententity.i (hunk 2 above)
Comment (hsorby) [reply src/bindings/interface/componententity.i]: 92dfea1

File: src/api/libcellml/componententity.h (hunk 1 above)
Comment (hsorby) [reply src/api/libcellml/componententity.h]: 92dfea1

**Review by agarny — DISMISSED** [review]

**Review by nickerso — DISMISSED** [review]

**Review by agarny — APPROVED** [review]

**Review by nickerso — APPROVED** [review]

**Pull request merged**